content/de/cloud_computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Vorlage für Definition
+status: Feedback erwünscht
+category: Konzept
+---
+
+## Was es ist
+Das Konzept, hinter dem Cloud Computing ist die Bereitstellung von Systemressourcen wie CPU, Netzwerk und Speicher auf Abruf und über das Internet.
+Das Cloud Computing ermöglicht den Zugriff und die Nutzung von Systemressourcen aus beliebigen Standorten.
+Diese Systemressourcen werden von Cloud-Anbietern wie AWS, GCP, Azure, Digitial Ocean, und anderen, aus unterschiedlichen Standorten, an den Endverbraucher vermietet.
+
+## Welches Problem es löst
+Die Skalierung von Systemressourcen stellt für viele Organisationen eine große Herausforderung dar.
+Organisationen müssen entweder fest verplante Systemressourcen bei bestehenden Rechenzentren einkaufen oder ein Rechenzentrum aufbauen und verwalten.
+Hiermit sind signifikante Planungsaufwände und Kosten verbunden.
+Beim Cloud Computing verlagert sich die Verantwortung für die Bereitstellung und den Betrieb von Systemressourcen an den Cloud-Anbieter.
```

Comment:
Ich bin mir nicht sicher ob das schon die Lösung des Problems beschreibt und somit in den nächsten Abschnitt gehört.

Reply:
In dem Teil wäre noch wichtig zu sagen, wie sich die Herausforderungen der Skalierung von den System Ressourcen positiv auf den Enduser auswirken durch das Cloud Computing, sprich alle Vorteile aufzählen

Reply:
Hier nochmal zum Vergleich die englische Fassung:

> Organizations traditionally faced two main problems when attempting to expand their use of computing power. They either acquire, support, design, and pay for facilities to host their physical servers and network or expand and maintain those facilities. Cloud computing allows organizations to outsource some portion of their computing needs to another organization.

Ich denke auch das hier ein Teil eigentlich schon erklärt wie das Problem gelöst wird. Vielleicht würde man das auch in der englischen Fassung auch in den nächsten Abschnitt verlagern?

>  Cloud computing allows organizations to outsource some portion of their computing needs to another organization.

@Petria3s weichen wir dann nicht schon zu stark vom englischen Original ab?

Reply:
Ich stimme @gdenn zu, wir sollten nicht zu sehr abweichen. Von daher passt der Content aus meiner Sicht so. Man könnte darüber nachdenken, den englischen Begriff zu verfeinern (und im Anschluss den deutschen nachziehen) aber ich halte es nicht für sooo wichtig.